src/Makefile

```diff
@@ -77,6 +81,16 @@ $(LIBDIR)/$(STATICLIBTARGET): $(LIBOBJ) $(DEVICELIB)
 	ar cr $@ $(LIBOBJ) $(TMP)/*.o
 	rm -Rf $(TMP)
 
+$(PKGDIR)/nccl.pc : nccl.pc.in
+	@mkdir -p $(PKGDIR)
+	printf "Generating %-35s > %s\n" $< $@
+	printf $(PREFIX)
```

Comment:
Not sure if that printf was supposed to stay

Reply:
oh, yes, is a typo

fixing